src/odemis/driver/test/hamamatsurx_test.py

```diff
@@ -603,6 +557,32 @@ def setUpClass(cls):
             if child.name == CONFIG_DELAYBOX["name"]:
                 cls.delaybox = child
 
+        cls.delaybox._metadata[model.MD_TIME_RANGE_TO_DELAY] = \
```

Comment:
Please use the API:
`cls.delaybox.updateMetadata({model.MD_TIME_RANGE_TO_DELAY: ....})`